lib/mixlib/shellout/windows.rb

```diff
@@ -183,7 +185,7 @@ def consume_output(open_streams, stdout_read, stderr_read)
 
```

Comment:
```suggestion
            # The unary plus operator (+) creates a mutable copy of the string returned by readpartial.
            # This is necessary because readpartial may return a frozen string, and we need to be able
            # to modify the string (append to buffers, manipulate encoding, etc.) in subsequent operations.
            # Without the +, attempting to modify a frozen string would raise a FrozenError.
```